ingestion/nse-scraper/src/main.py

```
@@ -0,0 +1,167 @@
+        raise typer.Exit(1)
+
+
+@app.command()
+def health() -> None:
+    """Check scraper health and dependencies."""
+    console.print("[bold]NSE Scraper Health Check[/bold]\n")
+
+    # Check Kafka connectivity
+    try:
+        from src.producers.avro_producer import AvroProducer
+        producer = AvroProducer(config.topics.raw_ohlc)
```

Ruff (0.14.10)

Local variable `producer` is assigned to but never used

Remove assignment to unused variable `producer`

(F841)